deployment/helm/synopsys-alert/README.md

```diff
@@ -25,34 +25,52 @@ Helm Charts for Synopsys Alert
 
 ## Installing the Chart -- Helm 2
 
+### Add the helm repository
+
+```bash
+ https://sig-repo.synopsys.com/artifactory/sig-cloudnative
```

Comment:
Should this also include the helm command? 
`helm repo add synopsys https://sig-repo.synopsys.com/artifactory/sig-cloudnative`

Reply:
yes